### PR TITLE
Fix tooltip position offset when charts zoomed in

### LINKED ISSUE #86: Tooltip doesn't show if the datapoint is off the chart
To reproduce:
In the drivers tab, modify the y-slider to a small number, then hover over the bars that are very long. no tooltip appears.

### PR DESCRIPTION
**Pull request by ktkization**
Fixes #86

### CODE REVIEW

File: webapp/lib/charts.dart [hunk 1]
```diff
@@ -4,6 +4,18 @@ import 'dart:html';
...
+
+init() {
+  chartjs.ChartTooltipPositioner custom = (List<dynamic> elements, chartjs.Point eventPosition) {
+    if (elements.isEmpty) {
+      return null;
+    }
+    return new chartjs.Point(x: eventPosition.x, y: eventPosition.y);
+  };
+  chartjs.ChartTooltipsStaticConfiguration()
+    ..positioners = custom;
```
Comment (marianamarasoiu) [inline webapp/lib/charts.dart]: Can you try:
```suggestion
  var tooltipConfig = chartjs.ChartTooltipsStaticConfiguration()
    ..positioners = {"custom": custom};
  chartjs.Chart.Tooltip = tooltipConfig;
```
?

Comment (ktkization) [reply webapp/lib/charts.dart]: changed this

File: webapp/lib/charts.dart [hunk 2]
```diff
@@ -4,6 +4,19 @@ import 'dart:html';
...
+init() {
+  chartjs.ChartTooltipPositioner custom = (List<dynamic> elements, chartjs.Point eventPosition) {
+    if (elements.isEmpty) {
+      return null;
+    }
+    return new chartjs.Point(x: eventPosition.x, y: eventPosition.y);
+  };
+  var tooltipConfig = chartjs.ChartTooltipsStaticConfiguration()
+    ..positioners = {'custom': custom};
+  chartjs.Chart.Tooltip = tooltipConfig;
```
Comment (marianamarasoiu) [inline webapp/lib/charts.dart]: ok, found the magic:
```suggestion
  chartjs.Chart.Tooltip.positioners.custom = custom;
```

**Review by marianamarasoiu — APPROVED** [review]
thanks @ktkization !

**Pull request merged**